packages/neon_lints/lib/src/base.yaml

```diff
@@ -209,7 +202,6 @@ linter:
     unnecessary_to_list_in_spreads: true
     unreachable_from_main: true
     unrelated_type_equality_checks: true
-    unsafe_html: true
```

Comment:
I did fix these issues, so we should be able to keep this rule?

Reply:
This one is actually deprecated and removed in dart 3.7

https://dart.dev/tools/linter-rules/unsafe_html

Reply:
Interesting, it was only added so recently :sweat_smile: